Day_1_Scientific_Python/04-scipy_statsmodels_introduction.ipynb

```diff
@@ -0,0 +1,1155 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<img width=700px; src=\"../img/logoUPSayPlusCDS_990.png\">\n",
+    "\n",
+    "<p style=\"margin-top: 3em; margin-bottom: 2em;\"><b><big><big><big><big>Introduction to Scipy</big></big></big></big></b></p>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. File input/output - `scipy.io`"
```

Comment:
Before starting with this, I would put a brief intro (could be just the sentence on this page: https://www.scipy.org/scipylib/index.html, or something from the readme https://github.com/scipy/scipy). 
At least a link to the docs maybe? (https://docs.scipy.org/doc/scipy/reference/)